dev-packages/node-integration-tests/suites/tracing/httpIntegration/test.ts

```diff
@@ -136,30 +168,246 @@ describe('httpIntegration', () => {
         await runner.completed();
       });
     });
-  });
 
-  test('allows to pass experimental config through to integration', async () => {
-    const runner = createRunner(__dirname, 'server-experimental.js')
-      .expect({
-        transaction: {
-          contexts: {
-            trace: {
-              span_id: expect.stringMatching(/[a-f0-9]{16}/),
-              trace_id: expect.stringMatching(/[a-f0-9]{32}/),
-              data: {
-                url: expect.stringMatching(/\/test$/),
-                'http.response.status_code': 200,
-                'http.server_name': 'sentry-test-server-name',
-              },
-              op: 'http.server',
-              status: 'ok',
-            },
-          },
+    describe('custom server.emit', () => {
```

Comment:
these tests have been added to verify that everything still works even if a user (or some other library) _also_ wraps `server.emit`. This was the case in the nestjs-distributed-tracing E2E test.